README.Rmd

```diff
@@ -99,16 +111,20 @@ emodnet_get_all_wfs_info()
 ```
 
 
-## Get WFS layers
+## Get data from a data source: get layers
 
 You can extract layers directly from a `wfs` object using layer names. 
 All layers are downloaded as `sf` objects and output as a list with a named element for each layer requested.
+The argument `reduce_layers = TRUE` stack all the layers in one single tibble, if possible (for instance if all column names are the same, otherwise it fails).
+
+By default, `emodnet_get_layers()` returns a list of sf objects, one per layer.
+
 
 ```{r}
 emodnet_get_layers(wfs = wfs_bio, layers = layers)
 ```
 
-You can change the output `crs` through the argument `crs`.
+You can change the output `crs` ("projection") through the argument `crs`.
```

Comment:
```suggestion
You can change the output Coordinate Reference System (CRS), which defines how geographic data is mapped to the Earth's surface,  through the argument `crs`.
```

CRS's can have projections (if they are projected) but you can also have geographic CRS's which are not projected, so projection is not quite accurate.

Reply:
I should read the basic resources myself